src/main/kotlin/no/nav/syfo/identendring/PdlAktorConsumer.kt

```diff
@@ -36,7 +38,17 @@ class PdlAktorConsumer(
                 try {
                     runConsumer()
                 } catch (ex: Exception) {
-                    log.error("Error running kafka consumer for pdl-aktor, unsubscribing and waiting $DELAY_ON_ERROR_SECONDS seconds for retry", ex)
+                    when (ex) {
+                        is InactiveIdentException -> {
+                            log.warn("New ident is inactive in PDL, unsubscribing and waiting $DELAY_ON_ERROR_SECONDS seconds for retry", ex)
```

Comment:
Spørs om dette skjer så mye, men tror kanskje vi egentlig burde logge error her?

Reply:
jeg tror egentlig dette er en error, for denne vil nok aldri løse seg selv :/